web-ui/src/components/reports-section/checkin-report/TeamMemberMap.jsx

```diff
@@ -44,33 +45,64 @@ const TeamMemberMap = ({ members, id, closed, planned, reportDate }) => {
                 className="team-member-map-accordion-summary"
               >
                 <Avatar
-                  className={'large'}
+                  sx={{ width: 54, height: 54 }}
                   src={getAvatarURL(member.workEmail)}
                 />
                 <div className="team-member-map-summmary-content">
-                  <Typography>{member.name}</Typography>
+                  <hgroup>
+                    <Typography variant="h6">{member.name}</Typography>
+                    <Typography sx={{ color: 'var(--muted)' }} variant="body2">
+                      {member.title}
+                    </Typography>
+                  </hgroup>
                   <Typography
                     variant="caption"
                     component={'time'}
                     dateTime={getLastCheckinDate(checkins).toISOString()}
-                    sx={{ display: { xs: 'none', sm: 'flex' } }}
+                    sx={{ display: { xs: 'none', sm: 'grid' } }}
                     className="team-member-map-summmary-latest-activity"
                   >
-                    {getLastCheckinDate(checkins).getFullYear() === 1969 ? (
-                      <p>No activity available.</p>
-                    ) : (
-                      <>
-                        Latest Activity:{' '}
-                        {getLastCheckinDate(checkins).toLocaleDateString(
-                          navigator.language,
-                          {
-                            year: 'numeric',
-                            month: '2-digit',
-                            day: 'numeric'
-                          }
-                        )}
-                      </>
-                    )}
+                    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+                      <Typography variant="overline" sx={{ mb: -1 }}>
+                        Check-In date:
+                      </Typography>
+                      {getCheckinDateForPeriod(
+                        checkins,
+                        reportDate
+                      ).getFullYear() === 1969 ? (
```

Comment:
I know you didn't add this bit, but I wonder what is special about the year 1969.

Reply:
Summer of Love? 🤷

Reply:
Updated to compare to epoch time at runtime, at not at my current timezone.